packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -192,7 +193,7 @@ export class Inserter extends Component {
 				{ renderToggle( {
 					onToggle: onPress,
 					isOpen,
-					disabled,
+					disabled: disabled || blockDoNotSupportInserter,
```

Comment:
I have concerns about bringing this because we're going to revert this after (https://github.com/WordPress/gutenberg/pull/20502) gets merged

Reply:
@pinarol wdyt ? It is temporary changes until support inserter PR is not merged.

 This affects all blocks and thats why I have concerns about iy

Reply:
Dropped comments below.

Reply:
resolved according to [this](https://github.com/WordPress/gutenberg/pull/19013/files#r403054817)